java/code/src/com/redhat/rhn/taskomatic/task/DailySummary.java

```diff
@@ -494,8 +494,8 @@ public String prepareEmail(
         args[5] = OrgFactory.EMAIL_FOOTER.getValue();
         args[6] = OrgFactory.EMAIL_ACCOUNT_INFO.getValue();
 
-        if (cloudPaygManager.isPaygInstance() && !cloudPaygManager.isCompliant()) {
-            args[7] = String.format("%s%n", ls.getMessage("dailysummary.email.notpaygcompliant"));
+        if (CLOUD_PAYG_MANAGER.isPaygInstance() && !CLOUD_PAYG_MANAGER.isCompliant()) {
+            args[7] = String.format("%s\n", ls.getMessage("dailysummary.email.notpaygcompliant"));
```

Comment:
SonarCloud chokes on this, use `%n` instead:
```suggestion
            args[7] = String.format("%s%n", ls.getMessage("dailysummary.email.notpaygcompliant"));
```

Reply:
> SonarCloud chokes on this, use `%n` instead:

@RohitBhojak this has  already been fixed in 
https://github.com/uyuni-project/uyuni/pull/10059
Are you sure you rebased correctly? My guess is that it passed unnoticed. Please fix

Reply:
I rebased correctly but I messed up in fixing the conflicts because I didn't notice the change. I will fix this ASAP.